src/ThreeEditor/Simulation/Scoring/ScoringOutput.ts

```diff
@@ -13,6 +14,8 @@ export type ScoringOutputJSON = Omit<
 	SimulationElementJSON & {
 		quantities: ScoringQuantityJSON[];
 		detectorUuid?: string;
+		zoneUuid?: string;
+		scoringType?: string;
```

Comment:
what about using some enum or const array, like here: https://github.com/yaptide/ui/blob/master/src/ThreeEditor/Simulation/Scoring/ScoringOutputTypes.ts#L133 ?
there are just two possible values of scoring type if I get the code correctly.